recipes/astyle/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "astyle" %}
+{% set version = "3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.5/astyle-{{ version }}.tar.bz2
```

This is required for the bot to work.

```suggestion
  url: https://downloads.sourceforge.net/project/astyle/astyle/astyle%20{{ version }}/astyle-{{ version }}.tar.bz2
```

---

I don't know how I missed this before, but this project also has source archive releases on GitLab with URLs that should be parseable by the autotick bot, so I've changed the URL over to there instead.